spec/javascripts/notificationsSpec.js

```diff
@@ -0,0 +1,56 @@
+describe("Notifications", function() {
+  beforeEach(function() {
+    var elements = [];
+    elements.push("<div id='notifications'></div>");
```

Comment:
In general, I'm wondering if there's a better way mock DOM. Any ideas @nchambe2 @piecedigital @maestromusica?